dev/tasks/homebrew-formulae/apache-arrow.rb

```diff
@@ -57,7 +57,8 @@ class ApacheArrow < Formula
   fails_with gcc: "5"
 
   def install
-    # https://github.com/Homebrew/homebrew-core/issues/76537
+    # This isn't for https://github.com/Homebrew/homebrew-core/issues/76537 .
```

Comment:
Is deleting the old comment enough?  Is there a reason a future reader would think this is related to 76537?